lib/plug/adapters/test/conn.ex

```diff
@@ -8,6 +8,10 @@ defmodule Plug.Adapters.Test.Conn do
     maybe_flush()
 
     uri = URI.parse(uri)
+    if is_binary(uri.path) && not String.starts_with?(uri.path, "/") do
```

Comment:
I needed to check if `uri.path` is binary because it is totally fine if it is `nil`

Reply:
You can use `and/2` here instead of `&&/2` since both sides are strictly boolean:

```suggestion
    if is_binary(uri.path) and not String.starts_with?(uri.path, "/") do
```